pkg/api/types.go

```diff
@@ -1566,6 +1566,11 @@ type PodSpec struct {
 	// If specified, the fully qualified Pod hostname will be "<hostname>.<subdomain>.<pod namespace>.svc.<cluster domain>".
 	// If not specified, the pod will not have a domainname at all.
 	Subdomain string `json:"subdomain,omitempty"`
+	//Optional: Docker "--shm-size" support. Defines the size of /dev/shm in the IPC namespace of the pod.
```

Comment:
@janosi -- to clarify, rather than default to the container runtime default shm size, we would define a Kubernetes default shm size.  This way, the API is consistent across any container runtime (rkt, docker, cri-o, hyper, etc.).  So I am asking that you remove "docker" from the documentation text.